src/tests/multihost/basic/conftest.py

```diff
@@ -40,12 +40,10 @@ def package_install(session_multihost):
                'openldap-clients krb5-workstation '\
                'sssd sssd-dbus sssd-kcm ' \
                'expect ldb-tools sssd-tools'
-    if 'Fedora' in distro:
-        cmd = 'dnf install -y %s' % (pkg_list)
-    elif '8.' in distro.split()[5]:
+    cmd = 'yum install -y %s' % (pkg_list)
```

Comment:
Its failing here for some distro (mainly fedora)

if '8.' in distro.split()[5]: